.idea/login_form.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all .ideas files should be ignored in .gitignore file. Search some articles about that

Reply:
remove .idea directory from the project :)